### ISSUE TITLE
Add clean and reset option

### ISSUE THREAD

**Issue by Stibbons** [Guake/guake#1074]
Like Gnome-terminal: clean and reset the complete content of the current terminal

**Comment by aichingm** [comment]
Do you mean something like destroying the terminal and creating a new one in its place?

**Comment by gsemet** [comment]
Hum we might loose the bash history. The reset today is a kind of fake scroll.

**Comment by aichingm** [comment]
Yes that's true, currently it behaves like Crtl+L or `clear`. But resetting the bash history could be a desired side effect (it would be like a hard reset).
